yt/visualization/color_maps.py

```diff
@@ -312,7 +312,7 @@ def make_colormap(ctuple_list, name=None, interpolate=True):
         If you wish this colormap to be added as a valid colormap to the
         current session, specify a name here.  Default: None
 
-    interpolation: boolean, optional
+    interpolate: boolean, optional
```

Comment:
let's fix the wording too ! "optional" should map to `typing.Optional`, meaning that "this can be `None`"
(I'm aware that `None` evaluates as False so technically `None` is a usable value here, but it's neither the default value nor a specific check, so there is no reason I can see to mark this as optional)
```suggestion
    interpolate: boolean
```